src/util/openfaas/index.js

```diff
@@ -64,6 +66,32 @@ const callWithRetry = async (
   }
 };
 
+const reconcileFunction = async (fnName) => {
+  logger.debug(`Reconciling faas function: ${fnName}`);
+
+  try {
+    const fns = await getFunctionList();
+
+    const lookup = find(fns, (entry) => entry.name === fnName);
```

Comment:
```suggestion
    const lookup = fns.find((entry) => entry.name === fnName);
```
is there advantage with lodash find